apps/publications/src/app/dialogs/add-category-dialog/add-category-dialog.component.ts

```diff
@@ -29,15 +29,16 @@ export class AddCategoryDialogComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.nameCtrl = new UntypedFormControl(null, [
+    this.nameCtrl = new FormControl('', [
       Validators.required,
       Validators.pattern('^[\\w.-]+( [\\w.-]+)*$'),
       Validators.maxLength(128),
     ]);
-    this.rankCtrl = new UntypedFormControl(null, [
+    this.rankCtrl = new FormControl(0, [
       Validators.required,
       Validators.pattern('^[0-9]+(\\.[0-9])?$'),
     ]);
+    this.rankCtrl.setValue(null);
```

Comment:
```suggestion
    this.rankCtrl = new FormControl<number>(null, [
      Validators.required,
      Validators.pattern('^[0-9]+(\\.[0-9])?$'),
    ]);
```

Reply:
If I try to create a typed FormControl with a null value, I get the ESLint error: `ESLint: Unsafe assignment of type FormControl<any> to a variable of type FormControl<number>.(@typescript-eslint/no-unsafe-assignment)`. For some reason, ESLint is OK with setting the null value but not when I create this typed object with the null value.

Reply:
Sorry, my bad, I tried it without `<number>` part, so it works well [fixed].